package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "platformsh-client",
-  "version": "0.1.191",
+  "version": "0.1.192",
```

Comment:
This version number will be increased when we release, can you revert that please?

Reply:
Oh that's right, thank you - reverted!